lua/starfall/libs_sv/prop.lua

```diff
@@ -73,11 +73,10 @@ function props_library.create(pos, ang, model, frozen)
 
 	local pos = SF.clampPos(vunwrap(pos))
 	local ang = aunwrap(ang)
-	model = SF.NormalizePath(model)
-	if not (util.IsValidModel(model) and util.IsValidProp(model)) then SF.Throw("Invalid model", 2) end
 
 	local ply = instance.player
-
+	model = SF.CheckModel(model, ply)
```

Comment:
💡 You probably want to call the `PlayerSpawnProp` hook here too

Reply:
It's on line 83